app/jobs/data_cleanup_job.rb

```diff
@@ -0,0 +1,18 @@
+class DataCleanupJob < ApplicationJob
+
+  queue_as :default
+
+  def perform
+    threshold = 90.days.ago
```

Comment:
This code ran fine in a staging console. I haven't tested it with SolidQueue yet, but figure that can happen on staging once this is merged into the `soomo-react-component` branch. Just want to be sure this looks right first.